lib/io.js

```diff
@@ -123,7 +123,16 @@ module.exports = app => {
   }
 
   if (config.redis) {
-    app.io.adapter(redis(config.redis));
+    // 增加对哨兵模式的支持
+    if (config.redis.sentinels) {
+      const Redis = require('ioredis');
```

Comment:
Contrast to require the ioredis driectly, I'd rather recommend choose egg-redis  as optional plugin dependency. Read more detail [here](https://eggjs.org/en/advanced/plugin.html).